### PR TITLE
Our second blog post - Ruby's secret team

### PR DESCRIPTION
**Pull request by MayarAlaa**

### CODE REVIEW

File: blog/_posts/2016-9-19-our-2nd-blog-post.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+layout: post
+created_at: 2016-09-19
+permalink: blog/2016-09-19-The-Miracle
+current: blog
+author: Nada Ashraf and Mayar Alaa
+twitter: RubysSecretteam
+---
+The Miracle <3
+
+![Ruby's Secret Team](/img/blog/2016/2016-09-19-RGSoC-dayoff.jpg.jpg){RGSoC-dayoff}
```
Comment (alicetragedy) [inline blog/_posts/2016-9-19-our-2nd-blog-post.md]: please add image credits to your post, as explained [here](https://github.com/rails-girls-summer-of-code/summer-of-code/blob/gh-pages/CONTRIBUTING.md#images).

File: blog/_posts/2016-9-19-our-2nd-blog-post.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+---
+title: The Miracle
+layout: post
+created_at: 2016-09-19
+permalink: blog/2016-09-19-The-Miracle
+current: blog
+author: Nada Ashraf and Mayar Alaa
+twitter: RubysSecretteam
```
Comment (alicetragedy) [inline blog/_posts/2016-9-19-our-2nd-blog-post.md]: You are missing the post category (it should be `student-posts`). Please add it to the metadata, following the same format as e.g. [here](https://raw.githubusercontent.com/rails-girls-summer-of-code/summer-of-code/gh-pages/blog/_posts/2016-08-24-hello-from-team-Crackers.md)

File: blog/_posts/2016-9-19-our-2nd-blog-post.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+---
+title: The Miracle
+layout: post
+created_at: 2016-09-19
+permalink: blog/2016-09-19-The-Miracle
```
Comment (alicetragedy) [inline blog/_posts/2016-9-19-our-2nd-blog-post.md]: To make it more consistent, it's best if your filename has the same title as your title/permalink. Can you rename the file to `2016-9-19-The-Miracle.md`?

**Review by alicetragedy — CHANGES_REQUESTED** [review]
Please change the things I've commented on before we can publish. You can ping me anytime if you have any trouble or specific questions. Good work! 👍

**Comment by MayarAlaa** [comment]
@alicetragedy Thank you so much !
I did the changes that you requested , is there anything else you want me to edit ?

**Review by alicetragedy — APPROVED** [review]
Great work, thank you @MayarAlaa!

**Comment by MayarAlaa** [comment]
@alicetragedy thank you so much

**Pull request merged**